slack as alternative in foundations 1

dexterchan opened the pull request:
Slack is free at any time at this moment. It is a battle tested notification app.

See if it works for you.

ed-donner commented:
Hey @dexterchan - this is great, but would you be able to make the change into the community_contributions folder? Also include a note that people should `uv add slack_sdk` but don't change the project dependencies if possible - thanks so much

dexterchan commented:
@ed-donner : done.... I changed the file to community contribution folder. reverted requirement.txt. Also, the notebook contain the line 
uv pip install slack_sdk==3.35.0
